arcgishub/sites.py

```diff
@@ -688,6 +714,8 @@ def search(self, title=None, owner=None, created=None, modified=None, tags=None)
         ---------------     --------------------------------------------------------------------
         title               Optional string. Return sites with provided string in title.
         ---------------     --------------------------------------------------------------------
+        url                 Optional string. Return sites with provided string in url.
```

Comment:
would there ever be multiple `sites` or just a single `site`?

Reply:
just a single site.